tests/console/check_system_info.pm

```diff
@@ -111,6 +121,8 @@ sub run {
         check_addons($myaddons);
         check_product("after");
         check_buildid;
+        # check the bzr to breezy after migration to sles15sp4
+        check_bzr_to_breezy if check_var('HDDVERSION', '15-SP4');
```

Comment:
Change HDDVERSION to VERSION should work.

Reply:
And is this only work on SLE15SP4? Or later?

Reply:
changed